### ISSUE TITLE
Not supporting the newly added "Verification property" in the Notion Wiki database

### ISSUE THREAD

**Issue by rkttu** [notion-dotnet/notion-sdk-net#368]
**Describe the bug**
A recent new Notion feature is the ability to turn the top of the document into a wiki database with a special type of database. Unlike other common databases, this adds a special attribute called "verification", which currently doesn't exist in Notion.Net, causing JSON interpretation errors.

**To Reproduce**
1. Turn one of the top articles into a wiki article.
2. Add the Notion app integration to any article in the wiki database.
3. Try calling the RetrieveAsync method with the page ID you added.
    for example, "await client.Pages.RetrieveAsync("...”);".
4. Newtonsoft.Json.JsonSerializationException throwed. "Error converting value \"verification\" to type 'Notion.Client.PropertyValueType'. Path

**Expected behavior**
The Notion.Client.PropertyValueType.Verification enumeration constant member must exist, and no JSON interpretation errors should appear at this stage.

**Screenshots**
<img width="1252" alt="스크린샷 2023-07-20 오후 3 18 41" src="https://github.com/notion-dotnet/notion-sdk-net/assets/1297346/3275aeb7-4380-439d-9f3c-c8324bea1d3a">

**Desktop:**
 - OS: macOS Ventura 13.4.1 (c)
 - Browser: Google Chrome 114.0.5735.198 (Official Build) (x86_64)
 - NuGet Version: Notion.Net 4.1.0 (via NuGet.org)
 - .NET Version: 7.0.304 (x86_64, macOS)

**Additional context**
I think you should look at other attributes as well, such as AI summarization.

With Notion promising big changes in the future, I expect to see even bigger changes in property types. Specifying property types as enumeration constants seems inefficient, as it requires Notion to release a package every time it adds a new property type. I'd like to see a more flexible way to do this.

**Comment by kirk-marple** [comment]
If anyone needs this, I created a fork, and added this class to get it working.

Have to add it as a known subtype in PropertyValue.cs, and add the enum:
```
    [JsonSubtypes.KnownSubTypeAttribute(typeof(VerificationPropertyValue), PropertyValueType.Verification)]
    
        [EnumMember(Value = "verification")]
        Verification

```

```
using Newtonsoft.Json;

namespace Notion.Client
{
    /// <summary>
    ///     Verification property value.
    /// </summary>
    public class VerificationPropertyValue : PropertyValue
    {
        public override PropertyValueType Type => PropertyValueType.Verification;

        [JsonProperty("verification")]
        public VerificationValue Verification { get; set; }
    }

    /// <summary>
    ///     Object containing verification type-specific data.
    /// </summary>
    public class VerificationValue
    {
        /// <summary>
        ///     The state of verification. Possible values are "verified" and "unverified".
        /// </summary>
        [JsonProperty("state")]
        public string State { get; set; }

        /// <summary>
        ///     Describes the user who verified this page.
        /// </summary>
        [JsonProperty("verified_by")]
        public User VerifiedBy { get; set; }

        /// <summary>
        ///     Date verification property values contain a date property value.
        /// </summary>
        [JsonProperty("date")]
        public DatePropertyValue Date { get; set; }
    }
}
```

**Comment by rkttu** [comment]
Seeing that it's been a while since I raised this issue and it still hasn't been reviewed, I think I'll check to see if the project is still in development.

**Comment by KoditkarVedant** [comment]
@rkttu I would still like to support this project but because of some work I couldn't spend time on it. I would try to see I can add support for this over a week.

**Comment by KoditkarVedant** [comment]
This has been released as part of [4.2.0](https://github.com/notion-dotnet/notion-sdk-net/releases/tag/4.2.0)
